sphinx/ext/intersphinx/_load.py

```diff
@@ -78,18 +70,18 @@ def validate_intersphinx_mapping(app: Sphinx, config: Config) -> None:
                 'Invalid value `%r` in intersphinx_mapping[%r]. '
                 'Expected a two-element tuple or list.'
             )
-            LOGGER.error(msg % (value, name))
+            LOGGER.error(msg, value, name)
             del config.intersphinx_mapping[name]
             continue
         try:
             uri, inv = value
-        except (TypeError, ValueError, Exception):
+        except Exception:
```

Comment:
The two first exceptions were un-necessary (also missed this one).

Reply:
Or alternatively handling only `TypeError` and `ValueError` -- the normal, natural cases -- and allowing other strange behaviours to bubble up as a genuine runtime exception.

Reply:
> Is the FakeList / NotImplementedError scenario valid given that value must be a tuple or list? (I suppose it could be a subclass of them...)

I took into account the fact that some people might create helpers in extensions to construct the intersphinx mapping. Since we anyway raise an exception at the end, it doesn't really matter whether we gobble it or not... But.. maybe you're right actually.

Reply:
EDIT: I think it's better to keep the large exception, otherwise I don't know what kind of message to output (I mean, it would mostly say the same...)

Reply:
See #12644.